src/cbmc/cbmc_parse_options.cpp

```diff
@@ -907,6 +907,8 @@ bool cbmc_parse_optionst::process_goto_program(
     // Similar removal of RTTI inspection:
     remove_instanceof(symbol_table, goto_functions);
     // Similar removal of java nondet statements:
+    // TODO Should really get this from java_bytecode_language somehow, but we
+    // don't have an instance of that here.
```

Comment:
A fix for this could be to pass the optionst reference through language_ui so that the options parsed there are also available here. I don't like that all these options are stored in separate variables in java_bytecode_languaget anyway.